p2p/client.go

```diff
@@ -38,7 +43,7 @@ func (s *peer) SendAnchoredDocument(ctx context.Context, receiverID identity.DID
 		if err != nil {
 			return nil, err
 		}
-		return h.SendAnchoredDocument(localCtx, in, receiverID)
+		return h.SendAnchoredDocument(localCtx, in, selfDID)
```

Comment:
Let's say I have 5 collaborators on a document. We loop over the 5 collaborators on a higher level but this method is called to actually send it to each of the receivers?

Or, I am wrong?

Reply:
A got it. It is the local account case.